srtcore/crypto.h

```diff
@@ -118,16 +118,26 @@ class CCryptoControl
 
     const unsigned char* getKmMsg_data(size_t ki) const { return m_SndKmMsg[ki].Msg; }
     size_t getKmMsg_size(size_t ki) const { return m_SndKmMsg[ki].MsgLen; }
-    bool getKmMsg_needSend(size_t ki) const
+    bool getKmMsg_needSend(size_t ki, bool runtime) const
```

Comment:
'runtime' is always false?

Reply:
The plan was to use these functions also in `CCryptoControl::sendKeysToPeer` and then move this whole function to `CUDT` in order to avoid reverse-call to `CUDT::sendSrtMsg` (another refactoring put aside). As this function is in `CCryptoControl`, it can use direct access to the "KmMsg" things instead of this function, but when moved to `CUDT` it will have to use them, and then the `runtime` parameter would have to be true. I could remove this parameter and associated code, but I really wouldn't like to lose it because if there happens next time the need to use them at the "during transmission" stage, the developer may forget the correct use of it in this situation and cause another hard to find bug.

Reply:
Sure.